internal/ui/settings_ui.go

```diff
@@ -378,6 +378,25 @@ func getPicturesDir() (fyne.ListableURI, error) {
 		return nil, err
 	}
 
-	uri := storage.NewFileURI(filepath.Join(home, "Pictures"))
-	return storage.ListerForURI(uri)
+	if runtime.GOOS == "darwin" {
+		uri, err := storage.Child(storage.NewFileURI(home), "Pictures")
+		if err != nil {
+			return nil, err
+		}
+
+		return storage.ListerForURI(uri)
+	}
+
+	const xdguserdir, pic = "xdg-user-dir", "PICTURES"
+	cmd := exec.Command(xdguserdir, pic)
+	stdout := bytes.NewBufferString("")
+	cmd.Stdout = stdout
+	if err = cmd.Run(); err != nil {
+		return nil, err
```

Comment:
This should fall back to a sensible default as well

Reply:
Good point.